guides/common/modules/proc_configuring-email-notification-preferences.adoc

```diff
@@ -1,12 +1,17 @@
-[id="Configuring_Email_Notifications_{context}"]
-= Configuring Email Notifications
+[id="Configuring_Email_Notification_Preferences_{context}"]
+= Configuring Email Notification Preferences
 
 You can configure {Project} to send email messages to individual users registered to {Project}.
 {Project} sends the email to the email address that has been added to the account, if present.
 Users can edit the email address by clicking on their name in the top-right of the {ProjectWebUI} and selecting **My account**.
 
 Configure email notifications for a user from the {ProjectWebUI}.
 
+[NOTE]
+====
+If you want to send email notifications to a group email address, instead of an individual email address, create a user account with the group email address and minimal {Project} permissions, then subscribe the user account to the desired notification types.
```

Comment:
```suggestion
If you want to send email notifications to a group email address instead of an individual email address, create a user account with the group email address and minimal {Project} permissions, then subscribe the user account to the desired notification types.
```